integreat_cms/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -118,22 +122,36 @@ <h1 class="heading">{% trans 'Technical Feedback' %}</h1>
                 {% endif %}
             </td>
         </tr>
-        {% endfor %}
-
-    </table>
-    <div class="flex flex-col">
-        {% include "pagination.html" with url=url chunk=admin_feedback %}
-        <div class="flex self-start gap-2 mt-2">
-            <select id="bulk-action">
-                <option>{% trans 'Select bulk action' %}</option>
-                <option data-bulk-action="{% url 'mark_admin_feedback_as_read' %}">{% trans 'Mark as read' %}</option>
-                <option data-bulk-action="{% url 'mark_admin_feedback_as_unread' %}">{% trans 'Mark as unread' %}</option>
-                {% if perms.cms.delete_feedback %}
-                    <option data-bulk-action="{% url 'delete_admin_feedback' %}">{% trans 'Delete' %}</option>
-                {% endif %}
-            </select>
-            <button id="bulk-action-execute" class="btn" disabled>{% trans 'Execute' %}</button>
+    {% endfor %}
+</table>
+{% if admin_feedback %}
+    <div class="pt-2 px-2">
+        <div class="inline">
+            <span class="text-gray-800 font-bold" data-admin-feedback-list-selection-count>0</span> <span class="text-gray-600">{% trans "Feedback selected" %}</span>
```

Comment:
```suggestion
            <span class="text-gray-800 font-bold" data-admin-feedback-list-selection-count>0</span> <span class="text-gray-600">{% trans "Feedback entries selected" %}</span>
```
Since "feedback" is an uncountable noun, e.g. the expression "4 feedback" does not make sense, I would append something like "entries" or similar...